src/Encoder.c

```diff
@@ -4706,4 +4706,30 @@ ZYDIS_EXPORT ZyanStatus ZydisEncoderDecodedInstructionToEncoderRequest(
     return ZYAN_STATUS_SUCCESS;
 }
 
+ZYDIS_EXPORT void ZydisEncoderNopFill(void *buffer, ZyanUSize length)
+{
```

Comment:
We should add the "invalid argument" checks here for consistency (`buffer == null`, `length < 0`).

Reply:
Length == 0 is a legitimate input for this function IMO. It should just do nothing in this case.

Reply:
Yes, but `length > 0` should error imho.

Reply:
This case is already eliminated by the argument being unsigned, unless I’m missing something.

Reply:
Ah yes, true. So just the buffer I guess :p